src/requirements.yml

```diff
@@ -20,12 +20,18 @@
   name: nvme
 - src: https://github.com/cisagov/ansible-role-persist-journald
   name: persist_journald
+- src: https://github.com/cisagov/ansible-role-pip
+  name: pip
 - src: https://github.com/cisagov/ansible-role-python
   name: python
+  version: bugfix/install-python3-minimal
+- src: https://github.com/cisagov/ansible-role-remove-python2
+  name: remove_python2
 - src: https://github.com/cisagov/ansible-role-ufw
   name: ufw
 - src: https://github.com/cisagov/ansible-role-upgrade
   name: upgrade
+  version: bugfix/update-cache-first
```

Comment:
There is an active PR for this branch right now. Do you plan on merging it once it's approved?

Reply:
The PR has been merged, so I have switched back to the default branch.